doc/adoc/user_guide.adoc

```diff
@@ -284,19 +301,25 @@ independent initrd can be created by setting
 build_host_independent_initrd: true|false
 
 == Run the Migration
+Migration can be triggered either via run_migration or via reboot.
+
+Option 1 - Running Migration via run_migration::
 After the install of the `SLES15-Migration` package, start the migration
-by calling the following command:
+using the install option by calling the following command:
```

Comment:
what is "install option" this seems to be misworded?

Reply:
maybe "start the migration process by calling.."

Reply:
In my original issue, the text got mangled.
https://github.com/SUSE/suse-migration-services/issues/267
It's supposed to point back to Section 4. "Install option 1" for run_migration.
or "Install option 2" for reboot.
i put a number sign 1 and it became a pointer to issue 1. not sure how to put a number sign. 
` Section 4. "Install option 1" for run_migration.
or "Install option 2" for reboot.
`

Reply:
fixed in the latest version
![option1_fixed](https://github.com/SUSE/suse-migration-services/assets/23247873/d5dae81c-93d4-4f81-b821-86cd6dcc1178)